openassessment/xblock/resolve_dates.py

```diff
@@ -24,16 +24,25 @@ class DateValidationError(Exception):
 DISTANT_FUTURE = dt.datetime(dt.MAXYEAR, 1, 1, tzinfo=pytz.utc)
 
 
-def get_current_time_zone(user_service):
+def get_user_preferences(user_service):
```

Comment:
@nasthagiri That's not a bad call. There isn't a strong candidate in the dir right now, but I could make it live alone in a 'user_info.py' file or something similar.